<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>purchaseWindow</class>
 <widget class="QMainWindow" name="purchaseWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>565</width>
    <height>434</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>楷体</family>
    <pointsize>13</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>商品采购中心</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../image/image.qrc">
    <normaloff>:/logo.png</normaloff>:/logo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 15px; /* 边框宽度 */
border-radius: 15px; /* 边框半径 */

border-top-style: solid;
border-top-width: 5px;
border-top-color: red;
border-top: 2px solid red;
 
border-right-style: solid;
border-right-width: 3px;
border-right-color: #55ff7f;
border-right: 2px solid #55ff7f;
 
border-bottom-style: solid;
border-bottom-width: 2px;
border-bottom-color: blue;
border-bottom:2px solid blue;
 
border-left-style: solid;
border-left-width: 3px;
border-left-color: aqua;
border-left: 2px solid #3ebaba;

background: aqua; /* 背景颜色 1E90FF*/
 font-family: 楷体; /* 设置字体 */
 color: write; /* 字体颜色 */
</string>
    </property>
    <property name="text">
     <string>商 品 采 购 中 心</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 15px; /* 边框宽度 */
border-radius: 20px; /* 边框半径 */

border-top-style: solid;
border-top-width: 5px;
border-top-color: red;
border-top: 1px solid red;
 
border-right-style: solid;
border-right-width: 3px;
border-right-color: #55ff7f;
border-right: 1px solid #55ff7f;
 
border-bottom-style: solid;
border-bottom-width: 2px;
border-bottom-color: blue;
border-bottom: 1px solid blue;
 
border-left-style: solid;
border-left-width: 3px;
border-left-color: aqua;
border-left: 1px solid aqua;

background:transparent; /* 背景颜色 1E90FF*/
 font-family: 楷体; /* 设置字体 */
 color: write; /* 字体颜色 */
</string>
    </property>
    <property name="text">
     <string>商品名称</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="nameEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="maxLength">
     <number>32768</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>50</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 15px; /* 边框宽度 */
border-radius: 20px; /* 边框半径 */

border-top-style: solid;
border-top-width: 5px;
border-top-color: red;
border-top: 1px solid red;
 
border-right-style: solid;
border-right-width: 3px;
border-right-color: #55ff7f;
border-right: 1px solid #55ff7f;
 
border-bottom-style: solid;
border-bottom-width: 2px;
border-bottom-color: blue;
border-bottom: 1px solid blue;
 
border-left-style: solid;
border-left-width: 3px;
border-left-color: aqua;
border-left: 1px solid aqua;

background:transparent; /* 背景颜色 1E90FF*/
 font-family: 楷体; /* 设置字体 */
 color: write; /* 字体颜色 */
</string>
    </property>
    <property name="text">
     <string>商品数量</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="quantityBox">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>50</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="inpriceEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 15px; /* 边框宽度 */
border-radius: 20px; /* 边框半径 */

border-top-style: solid;
border-top-width: 5px;
border-top-color: red;
border-top: 1px solid red;
 
border-right-style: solid;
border-right-width: 3px;
border-right-color: #55ff7f;
border-right: 1px solid #55ff7f;
 
border-bottom-style: solid;
border-bottom-width: 2px;
border-bottom-color: blue;
border-bottom: 1px solid blue;
 
border-left-style: solid;
border-left-width: 3px;
border-left-color: aqua;
border-left: 1px solid aqua;

background:transparent; /* 背景颜色 1E90FF*/
 font-family: 楷体; /* 设置字体 */
 color: write; /* 字体颜色 */
</string>
    </property>
    <property name="text">
     <string>进货价</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>70</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 15px; /* 边框宽度 */
border-radius: 20px; /* 边框半径 */

border-top-style: solid;
border-top-width: 5px;
border-top-color: red;
border-top: 1px solid red;
 
border-right-style: solid;
border-right-width: 3px;
border-right-color: #55ff7f;
border-right: 1px solid #55ff7f;
 
border-bottom-style: solid;
border-bottom-width: 2px;
border-bottom-color: blue;
border-bottom: 1px solid blue;
 
border-left-style: solid;
border-left-width: 3px;
border-left-color: aqua;
border-left: 1px solid aqua;

background:transparent; /* 背景颜色 1E90FF*/
 font-family: 楷体; /* 设置字体 */
 color: write; /* 字体颜色 */
</string>
    </property>
    <property name="text">
     <string>出售价</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="outpriceEdit">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>70</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>50</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton { /* 按钮美化 */
 width: 50px; /* 宽度 */
 height: 50px; /* 高度 */
 border-width:3px; /* 边框宽度 */
 border-radius: 20px; /* 边框半径 */
 border-style: outset; 
 border-color:#00aaff;
 background: aqua; /* 背景颜色 1E90FF*/
 color: black; /* 字体颜色 */

}
QPushButton:hover { /*5599FF 鼠标移入按钮范围时改变颜色 */
 background: #00d9d9;
 border-style: inset;
 color: black; /* 字体颜色 */
}</string>
    </property>
    <property name="text">
     <string>添加</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>461</width>
      <height>241</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_5">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 15px; /* 边框宽度 */
border-radius: 20px; /* 边框半径 */

border-top-style: solid;
border-top-width: 5px;
border-top-color: red;
border-top: 1px solid red;
 
border-right-style: solid;
border-right-width: 3px;
border-right-color: #55ff7f;
border-right: 1px solid #55ff7f;
 
border-bottom-style: solid;
border-bottom-width: 2px;
border-bottom-color: blue;
border-bottom: 1px solid blue;
 
border-left-style: solid;
border-left-width: 3px;
border-left-color: aqua;
border-left: 1px solid aqua;

background:transparent; /* 背景颜色 1E90FF*/
 font-family: 楷体; /* 设置字体 */
 color: write; /* 字体颜色 */
</string>
    </property>
    <property name="text">
     <string>序</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_6">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>100</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 15px; /* 边框宽度 */
border-radius: 20px; /* 边框半径 */

border-top-style: solid;
border-top-width: 5px;
border-top-color: red;
border-top: 1px solid red;
 
border-right-style: solid;
border-right-width: 3px;
border-right-color: #55ff7f;
border-right: 1px solid #55ff7f;
 
border-bottom-style: solid;
border-bottom-width: 2px;
border-bottom-color: blue;
border-bottom: 1px solid blue;
 
border-left-style: solid;
border-left-width: 3px;
border-left-color: aqua;
border-left: 1px solid aqua;

background:transparent; /* 背景颜色 1E90FF*/
 font-family: 楷体; /* 设置字体 */
 color: write; /* 字体颜色 */
</string>
    </property>
    <property name="text">
     <string>商品名称</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_7">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>100</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 15px; /* 边框宽度 */
border-radius: 20px; /* 边框半径 */

border-top-style: solid;
border-top-width: 5px;
border-top-color: red;
border-top: 1px solid red;
 
border-right-style: solid;
border-right-width: 3px;
border-right-color: #55ff7f;
border-right: 1px solid #55ff7f;
 
border-bottom-style: solid;
border-bottom-width: 2px;
border-bottom-color: blue;
border-bottom: 1px solid blue;
 
border-left-style: solid;
border-left-width: 3px;
border-left-color: aqua;
border-left: 1px solid aqua;

background:transparent; /* 背景颜色 1E90FF*/
 font-family: 楷体; /* 设置字体 */
 color: write; /* 字体颜色 */
</string>
    </property>
    <property name="text">
     <string>数量</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_8">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>100</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 15px; /* 边框宽度 */
border-radius: 20px; /* 边框半径 */

border-top-style: solid;
border-top-width: 5px;
border-top-color: red;
border-top: 1px solid red;
 
border-right-style: solid;
border-right-width: 3px;
border-right-color: #55ff7f;
border-right: 1px solid #55ff7f;
 
border-bottom-style: solid;
border-bottom-width: 2px;
border-bottom-color: blue;
border-bottom: 1px solid blue;
 
border-left-style: solid;
border-left-width: 3px;
border-left-color: aqua;
border-left: 1px solid aqua;

background:transparent; /* 背景颜色 1E90FF*/
 font-family: 楷体; /* 设置字体 */
 color: write; /* 字体颜色 */
</string>
    </property>
    <property name="text">
     <string>进货价</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_9">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>100</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 15px; /* 边框宽度 */
border-radius: 20px; /* 边框半径 */

border-top-style: solid;
border-top-width: 5px;
border-top-color: red;
border-top: 1px solid red;
 
border-right-style: solid;
border-right-width: 3px;
border-right-color: #55ff7f;
border-right: 1px solid #55ff7f;
 
border-bottom-style: solid;
border-bottom-width: 2px;
border-bottom-color: blue;
border-bottom: 1px solid blue;
 
border-left-style: solid;
border-left-width: 3px;
border-left-color: aqua;
border-left: 1px solid aqua;

background:transparent; /* 背景颜色 1E90FF*/
 font-family: 楷体; /* 设置字体 */
 color: write; /* 字体颜色 */
</string>
    </property>
    <property name="text">
     <string>出售价</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_10">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>100</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 15px; /* 边框宽度 */
border-radius: 20px; /* 边框半径 */

border-top-style: solid;
border-top-width: 5px;
border-top-color: red;
border-top: 1px solid red;
 
border-right-style: solid;
border-right-width: 3px;
border-right-color: #55ff7f;
border-right: 1px solid #55ff7f;
 
border-bottom-style: solid;
border-bottom-width: 2px;
border-bottom-color: blue;
border-bottom: 1px solid blue;
 
border-left-style: solid;
border-left-width: 3px;
border-left-color: aqua;
border-left: 1px solid aqua;

background:transparent; /* 背景颜色 1E90FF*/
 font-family: 楷体; /* 设置字体 */
 color: write; /* 字体颜色 */
</string>
    </property>
    <property name="text">
     <string>进货总价</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="insertButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>360</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton { /* 按钮美化 */
 width: 50px; /* 宽度 */
 height: 50px; /* 高度 */
 border-width:3px; /* 边框宽度 */
 border-radius: 20px; /* 边框半径 */
 border-style: outset; 
 border-color:#00aaff;
 background: aqua; /* 背景颜色 1E90FF*/
 color: black; /* 字体颜色 */

}
QPushButton:hover { /*5599FF 鼠标移入按钮范围时改变颜色 */
 background: #00d9d9;
 border-style: inset;
 color: black; /* 字体颜色 */
}</string>
    </property>
    <property name="text">
     <string>保存入库</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>370</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>总金额</string>
    </property>
   </widget>
   <widget class="QLabel" name="moneylabel">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>370</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>420</y>
      <width>321</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
     </font>
    </property>
    <property name="text">
     <string>注：点击右上角&quot;x&quot;可以返回上一界面</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../image/image.qrc"/>
 </resources>
 <connections/>
</ui>
